src/test/java/com/exasol/containers/ExasolContainerTest.java

```diff
@@ -83,15 +82,16 @@ void testWithExposedPorts() {
     @Test
     void testWithDefaultExposedPorts() {
         try (final ExasolContainer<?> container = new ExasolContainer<>()) {
-            assertThat(container.getExposedPorts().size(), equalTo(2));
+            assertThat(container.getExposedPorts().size(), equalTo(3));
         }
     }
 
     @Test
     void testAddExposedPorts() {
         try (final ExasolContainer<?> container = new ExasolContainer<>()) {
+            final int countBefore = container.getExposedPorts().size();
             container.addExposedPorts(1000);
-            assertThat(container.getExposedPorts().size(), equalTo(3));
+            assertThat(container.getExposedPorts().size(), equalTo(countBefore + 1));
```

Comment:
maybe you can also check that it contains the added port?

Reply:
✅